app/config/config.yml

```diff
@@ -191,6 +197,20 @@ doctrine:
 doctrine_migrations:
     dir_name: "%kernel.root_dir%/DoctrineMigrations"
 
+# Swiftmailer Configuration
+swiftmailer:
+    spool:     { type: memory }
+    transport: "%mailer_transport%"
+    host:      "%mailer_host%"
+    username:  "%mailer_user%"
+    password:  "%mailer_password%"
+    # using an env var throws an error
+    # auth_mode: "%mailer_auth_mode%"
+    auth_mode: cram-md5
+    port:      "%mailer_port%"
+    # to disable mail uncomment MAILER_URL
+    # MAILER_URL: "null://localhost"
```

Comment:
This does not appear true for Symfony 3.3/3.4 (there is a change in 4.0).

https://symfony.com/doc/3.4/email.html indicates a `disable_delivery` attribute can be used.